datafusion/src/physical_plan/functions.rs

```diff
@@ -3611,17 +3607,19 @@ mod tests {
         Ok(())
     }
 
-    #[test]
-    fn test_concat_error() -> Result<()> {
```

Comment:
This error is now thrown during execution instead of logical plan. I can fix the test case to check execution error if required.

Reply:
I think it would be valuable to update the test case

Reply:
Done 👍